### PR TITLE
chore(release): idem v2026.5.5

### PR DESCRIPTION
**Pull request by ig-impact**
## Release issue

Closes #13

## Summary

Release `idem v2026.5.5` — brief description of what this release brings.

## Reviewer checklist

**Code & tests**

- [ ] R CMD check — 0 errors, 0 warnings, 0 notes (`devtools::check()`)
- [ ] Full test suite passes (`devtools::test()`)
- [ ] No broken URLs (`urlchecker::url_check()`)

**Documentation**

- [ ] `NEWS.md` — entry is accurate, covers all user-facing changes, written from the user's perspective (not commit-log style)
- [ ] `man/` — all exported functions documented; examples are runnable
- [ ] `README.md` — built from `README.Rmd` (`devtools::build_readme()`), installation tag points to `v2026.5.5`
- [ ] pkgdown site builds without errors (`pkgdown::build_site()`)

**Package metadata**

- [ ] `DESCRIPTION` — `Version:` matches the PR title

**If this PR includes code changes**

- [ ] New and changed behaviour is covered by tests; edge cases are included
- [ ] Test coverage has not regressed
- [ ] New exported functions have complete roxygen documentation with runnable examples
- [ ] No example is wrapped in `\dontrun{}` unless it genuinely cannot run in the check environment

**Merge**

- [ ] Merge strategy — **merge commit, not squash** (preserves individual conventional commits for the next changelog)

### CODE REVIEW

File: inst/WORDLIST [hunk 1]
```diff
@@ -78,8 +78,11 @@ eec
 eed
 eeec
 ef
+endfor
+endif
 english
 env
+erge
```
[the remark this answers is not in the document]
Comment (ig-impact) [reply inst/WORDLIST:85]: not important.

File: justfile [hunk 2]
```diff
@@ -79,6 +79,12 @@ check-form:
     Rscript tools/trim_xlsform.R
     uv run python tools/validate_xlsform.py inst/extdata/form_required.xlsx
 
+# ── Release ───────────────────────────────────────────────────────────────────
+
+# Prepend a NEWS.md entry for VERSION from unreleased commits (e.g. `just news 2026.5.5`)
+news version:
+    uvx git-cliff --unreleased --tag v{{ version }} --prepend NEWS.md
```
[the remark this answers is not in the document]
Comment (ig-impact) [reply justfile:86]: made a commit to use uvx, in line with the contribution guide installing uv .
